website/pages/docs/nia/architecture.mdx

```diff
@@ -6,4 +6,38 @@ description: >-
   Consul-Terraform-Sync Architecture
 ---
 
-# Consul-Terraform-Sync Architecture
+# Consul Terraform Sync Architecture
+
+Consul Terraform Sync is a service-oriented tool for managing
+network infrastructure near real-time. Consul-Terraform-Sync runs as a daemon
+and integrates the network topology maintained by your Consul cluster with your
+network infrastructure to dynamically secure and connect services.
+
+## 10,000 Foot View
+
+[![Consul Terraform Sync Architecture](/img/nia-highlevel-diagram.png)](/img/nia-highlevel-diagram.png)
```

Comment:
This image isn't loading for me in the preview, would you have any idea why?

Reply:
here's the image for reference: https://deploy-preview-8732--consul-docs-preview.netlify.app/img/nia-highlevel-diagram.png I think it's because it's because it's on the integrations program doc which is on a separate branch

Reply:
Anyone know if there are any plans for that branch to be merged or should I add it to this PR?

Reply:
@BeenzSyed told me in chat that it was destined for inclusion. So I don't need to add it here.

We should probably remember to double check that it is cool once we've merged these things together.